llvm/lib/Transforms/IPO/MemProfContextDisambiguation.cpp

```diff
@@ -5050,6 +5050,45 @@ bool MemProfContextDisambiguation::initializeIndirectCallPromotionInfo(
   return true;
 }
 
+#ifndef NDEBUG
```

Comment:
I think we can drop the ifndef around the function definition in favour of a LLVM_ATTRIBUTE_UNUSED function attribute?

Reply:
I could, but then I need to put that attribute back on the StackIdIndexIter def in this code that I had removed it from. My sense is that the `#ifdef` is simpler overall, wdyt?

Reply:
Sounds good to me